src/DbTests/facil.yaml

```diff
@@ -192,6 +192,9 @@ rulesets:
           tvp:
             type: dbo.SingleColNonNull
 
+      - for: MenyColumns.sql
+        result: nominal
```

Comment:
Does this only happen with `nominal`?

Reply:
This also happens with the anonymous type